spec/book_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+require_relative '../person'
+require_relative '../book'
+require_relative '../rental'
+require_relative '../teacher'
+require_relative '../student'
+
+describe 'Book' do
+  before :each do
```

Comment:
Please add tests for all the methods of the `Book` class.

Reply:
Thank you for letting us know. We added all the tests for the Book class.